docs/content/usage/packages/helm.en-us.md

```diff
@@ -36,13 +35,15 @@ helm repo add  --username {username} --password {password} {repo} https://gitea.
 helm cm-push ./{chart_file}.tgz {repo}
 ```
 
-| Parameter    | Description |
-| ------------ | ----------- |
-| `username`   | Your Gitea username. |
-| `password`   | Your Gitea password. If you are using 2FA or OAuth use a [personal access token](development/api-usage.md#authentication) instead of the password. |
-| `repo`       | The name for the repository. |
-| `chart_file` | The Helm Chart archive. |
-| `owner`      | The owner of the package. |
+| Parameter      | Description                                                                                                                                    |
+| -------------- | ---------------------------------------------------------------------------------------------------------------------------------------------- |
+| `username`   | Your Gitea username.                                                                                                                           |
+| `password`   | Your Gitea password. If you are using 2FA or OAuth use a[personal access token](development/api-usage.md#authentication) instead of the password. |
```

Comment:
```suggestion
| `password`   | Your Gitea password. If you are using 2FA or OAuth use a [personal access token](development/api-usage.md#authentication) instead of the password. |
```